env_setup/env_setup.yml

```diff
@@ -65,31 +68,8 @@
       when: esxi_version is version('8.0.0', '>=')
 
     - name: "Cleanup snapshots and get VM related information for existing VM"
-      when: vm_exists is defined and vm_exists
-      block:
-        - name: "Not cleanup old snapshot by default"
-          ansible.builtin.set_fact:
-            cleanup_old_snapshots: false
-          when: cleanup_old_snapshots is undefined
-
-        - name: "Revert and cleanup existing snapshots on existing VM if required"
-          include_tasks: ../common/vm_cleanup_snapshot.yml
-          when: cleanup_old_snapshots | bool
-
-        - name: "Get existing VM info"
-          include_tasks: ../common/vm_get_vm_info.yml
-
-        - name: "Get VM's power state"
-          include_tasks: ../common/vm_get_power_state.yml
-
-        - name: "Power on VM"
-          include_tasks: ../common/vm_set_power_state.yml
-          vars:
-            vm_power_state_set: "powered-on"
-          when: vm_power_state_get == "poweredOff"
-
-        - name: "Wait for VM primary network adapter has MAC address"
-          include_tasks: ../common/vm_wait_primary_nic_mac.yml
+      include_tasks: handle_existing_vm.yml
```

Comment:
Using `handle` looks like there is some problem with the VM. How about changing the file name to `setup_existing_vm.yml` or `existing_vm_setup.yml`?

Reply:
Changed, thanks.